From ea3139a9002ba0061a9c9bc805d91a85704239ed Mon Sep 17 00:00:00 2001
From: sebastian suesens <sebastian.suesens@baslerweb.com>
Date: Tue, 14 May 2019 13:27:10 +0200
Subject: [PATCH 04/20] [imx8mm, imx8mq] increase max video memory from 64 to
 128MB

4 video buffers are needed to get 13MP@21fps
without the patch only to 2 buffers at 13MP are supported
---
 drivers/media/platform/mxc/capture/mx6s_capture.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/mxc/capture/mx6s_capture.c b/drivers/media/platform/mxc/capture/mx6s_capture.c
index 19d65de50..340bbb8e2 100644
--- a/drivers/media/platform/mxc/capture/mx6s_capture.c
+++ b/drivers/media/platform/mxc/capture/mx6s_capture.c
@@ -50,7 +50,7 @@
 #define MX6S_CAM_VERSION "0.0.1"
 #define MX6S_CAM_DRIVER_DESCRIPTION "i.MX6S_CSI"
 
-#define MAX_VIDEO_MEM 64
+#define MAX_VIDEO_MEM 128
 
 /* reset values */
 #define CSICR1_RESET_VAL	0x40000800
-- 
2.34.1

